### ISSUE TITLE
what does the "allow secondary searches" option do?

### ISSUE THREAD

**Issue by grenzionky** [FooSoft/yomichan#167]
i cant find any documentation on it on github or https://foosoft.net/projects/yomichan/, and pressing shift on the popup (my scan modifier key) doesnt make another popup appear.

**Comment by siikamiika** [comment]
TLDR you probably wanted to know about these issues #139 #75 instead, but I'll explain below what "allow secondary searches" actually does

-----------

It was added together with a feature that I was implementing a few years ago. Sorry that it's so confusing :blush:

Basically, if you have two dictionaries called Dictionary 1 and Dictionary 2 containing definitions for words, set result grouping to "Group results by main dictionary entry", set "Main dictionary for merged mode" to Dictionary 1 and click "Allow secondary searches" on Dictionary 2, Yomichan will do a secondary lookup from Dictionary 2 using Dictionary 1 headwords after doing the initial lookup from Dictionary 1 that fetches the grouped definition.

With actual examples, substituting Dictionary 1 with JMDict and Dictionary 2 with 明鏡国語辞典:

1. JMDict is set as the main dictionary but secondary searches from 明鏡国語辞典 are not enabled, and "**に**" is looked up
![2019-07-29_02-47-51](https://user-images.githubusercontent.com/5602483/62014575-5b40ad00-b1ab-11e9-95a9-52adbcd3c0f0.png)
The only result from 明鏡国語辞典 is read に although some of the headwords are read "**ふた**".
![2019-07-29_02-35-25](https://user-images.githubusercontent.com/5602483/62014580-67c50580-b1ab-11e9-8ad0-cc9daa0eb248.png)
2. JMDict is set as the main dictionary, secondary searches from 明鏡国語辞典 are enabled, and "**に**" is looked up
![2019-07-29_02-35-46](https://user-images.githubusercontent.com/5602483/62014636-61835900-b1ac-11e9-9925-c16bebbf9dc7.png)
ニ, ふた is there!
![2019-07-29_02-37-26](https://user-images.githubusercontent.com/5602483/62014638-6811d080-b1ac-11e9-94e9-7536319a73e8.png)

I noticed that the secondary search checkbox appears even without enabling result grouping by main dictionary, and it could be hidden in my opinion to avoid confusion. Then again, the checkbox doesn't appear without enabling advanced options, so the user is expected to know what they are doing.

**Comment by grenzionky** [comment]
thats actually makes a lot of sense. thank you
